messages/teleporter/message_manager_test.go

```diff
@@ -49,6 +49,17 @@ var (
 	}
 )
 
+func getChainIDBytes(id string) ([32]byte, error) {
+	chainID, err := ids.FromString(id)
+	if err != nil {
+		return [32]byte{}, err
+	}
+	chainIDBytes := common.Hex2Bytes(chainID.Hex())
+	var array32 [32]byte
+	copy(array32[:], chainIDBytes)
+	return array32, nil
+}
+
 func TestShouldSendMessage(t *testing.T) {
```

Comment:
In this test, let's use `PackTeleporterMessage` instead of `PackSendCrossChainMessageEvent`

Reply:
Fixed.